AFMReader/jpk.py

```diff
@@ -128,6 +155,8 @@ def load_jpk(file_path: Path | str, channel: str, config_path: Path | str | None
     config_path : Path | str | None
         Path to a configuration file. If ''None'' (default) then the packages default configuration is loaded from
         ''default_config.yaml''.
+    flip_image : bool, optional
+        Whether to flip the image vertically. Default is True.
```

Comment:
[ReStructured Text]() formatting...

```suggestion
        Whether to flip the image vertically. Default is ``True``.
```